lua/entities/gmod_tardis/modules/sh_timer.lua

```diff
@@ -0,0 +1,29 @@
+local timers = {}
```

Comment:
This should definitely be initialised as self.timers on an Initialize hook (or maybe pre-initialize) or this will conflict with other TARDISes

Reply:
Fixed

Reply:
`ENT.timers` does work though (without any hooks)
I've checked that different TARDISes get separate hooks now